source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,57 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes of data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. The ``watch()`` method takes an optional pipeline as the
```

Comment:
Should this sentence be in the second paragraph since it's a parameter (using the Method-Options-Return-Error structure)?

Reply:
Fixed